src/components/Icons/Arrows/ArrowChevronDoubleUp.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import { IconProps } from '../types';
+import BaseIcon from '../BaseIcon';
+import { getFill } from '../utils';
+
+function ArrowChevronDoubleUp({
+  id = 'ArrowChevronDoubleUptIcon',
```

Comment:
Shouldn't it be `ArrowChevronDoubleUpIcon` ?
```suggestion
  id = 'ArrowChevronDoubleUpIcon',
```

Reply:
comment resolved in https://github.com/magnetis/astro-native/pull/538/commits/c88d626f3df6d91b787089916a173d70b2b52cce